Lesson30/Firat/my-app/src/components/Exercise10/index.jsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+export const Exercise10 = () => {
+  return (
+    <div>
+      <button
+        onDoubleClick={() => {
```

Comment:
There's no 'onDoubleClick' in react. Instead, you'll have to check `event.detail` property. 

```
function handleClick (event) {
  if (event.detail == 2) alert('Double clicked!');
}
.....
        onClick={handleClick}
```